differ/differ.go

```diff
@@ -385,6 +388,9 @@ func (differ *FilesDiffer) diffSorted() (map[uint32][]string, map[uint32][]strin
 			file1Len := len(differ.file1.sortedEntries[srcColId])
 			file2Len := len(differ.file2.sortedEntries[tgtColId])
 
+			differ.file1ItemCount = file1Len
```

Comment:
Is this accurate? It doesn't look accurate to me. 
The double-nested for loop is technically going through each source Collection ID and target collection ID mapping. For example, if you have explicit mapping:
```
s1.col1 -> s1t.col1t
s1.col2 -> s1t.col2t
```
Then the for loop would go through two times to diff between each of the mapping.
The first time for loop executes, `file1ItemCount` will be the the number of items in `s1.col1`, and `file2ItemCount` will be the number of items in `s1.col1t`.
The second time for loop executes, `file1ItemCount` will be overwritten with the number of items in `s1.col2` instead, and same situation for the target `s1.col2t`.

You need to also watch out that the numbers work out for migration mode. For migration mode, one source collection can replicate to multiple target collections based on the filter. So the logic needs to not double count. 
At this point, it may be easier to skip the counting logic (don't use `fileXItemCount`) if `len(tgtColIds) > 0` as that would indicate migration mode.

Reply:
Another thing - if explicit mapping was not specified for a specific collection, then the items won't be diffed and the count of items per bucket won't be accurate

Reply:
I have updated this.